### ISSUE TITLE
sink and source

### ISSUE THREAD

**Issue by MoinRajpoot** [LDmicro/LDmicro#197]
can we use arduino  with sink and source options 
for input and output as normally arduino interface in market use sink (low level trigger) option

**Comment by TecnoCalubert** [comment]
I do not understand your question
No entiendo bien tu pregunta

**Comment by MoinRajpoot** [comment]
Can we use Ld micro to program a atmega 328 with low level input trigger
and low level output trigger

**Comment by TecnoCalubert** [comment]
Yes LDMicro The internal resistors of the Atmega328 can be selected
Si LDMicro Se puede seleccionar las resistencias internas del Atemega328

**Comment by MoinRajpoot** [comment]
How can we do that

**Comment by TecnoCalubert** [comment]
There I set the example for a doubt look at it
http://cq.cx/ladder-forum.pl?action=viewthread&parent=8955&tt=1566986668

**Comment by MoinRajpoot** [comment]
OK thanks dear

**Comment by TecnoCalubert** [comment]
Tell me if you understood
Dime si lo entendiste

**Comment by LDmicroGitHub** [comment]
There is a shorter way.  
LD file at the forum https://cq.cx/ladder-forum.pl?action=viewthread&parent=8955  
![image](https://user-images.githubusercontent.com/48811061/110224958-1b928900-7ee9-11eb-95e1-9a540a6aa364.png)
